src/index.ts

```diff
@@ -183,6 +183,25 @@ export default function PluginVue(userOptions: Partial<Options> = {}): Plugin {
         } else if (query.type === 'style') {
           debug(`transform(${id})`)
           const block = descriptor.styles[query.index]!
+
+          let preprocessOptions = options.preprocessOptions || {}
+          const preprocessLang = (options.preprocessStyles
+            ? block.lang
+            : undefined) as SFCAsyncStyleCompileOptions['preprocessLang']
+
+          if (preprocessLang) {
```

Comment:
We can probably avoid making this a breaking change by checking `preprocessLang in preprocessOptions`, and if it doesn't exist, fallback to using the top level `preprocessOptions` for compatibility with the old behavior.

Reply:
Is it like this ?